cargo-afl/tests/integration.rs

```diff
@@ -54,6 +59,32 @@ fn integration_cfg() {
     }
 }
 
+#[test]
+fn integration_maze() {
+    if !common::plugins_available().unwrap_or_default() {
+        #[allow(clippy::explicit_write)]
+        writeln!(
+            std::io::stderr(),
+            "Skipping `integration_maze` test as plugins are unavailable"
+        )
+        .unwrap();
+        return;
+    }
+
+    assert_cmd::Command::new(cargo_afl_path())
+        .arg("afl")
+        .arg("build")
+        .arg("-r")
```

Comment:
I think this may be the problem.

`target_dir_path` assumes the target binary is in the `target/debug` directory: https://github.com/rust-fuzz/afl.rs/blob/88176755ef11e77f4cbd340529b3dec4a6b875e7/cargo-afl/tests/integration.rs#L11-L19

Reply:
I think the example is getting built here: https://github.com/rust-fuzz/afl.rs/blob/88176755ef11e77f4cbd340529b3dec4a6b875e7/.github/workflows/rust.yml#L93

But I take your point, this doesn't completely answer the question.